### PR TITLE
fix(docs): update references table

### PR DESCRIPTION
**Pull request by ndom91**
## ☕️ Reasoning

- References updates to general integrations 

## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Comment by ndom91** [comment]
@balazsorban44 what do you think? Should it stay as `/references`?

File: docs/docs/reference/index.md [hunk 1]
```diff
@@ -16,11 +16,12 @@ Here are the _state_ of planned and released packages under the `@auth/*` and `@
...
 | `@auth/express`        | Released (experimental). See [docs](/reference/express)                                                                                                                                |
 | `@auth/remix`          | Planned.                                                                                                                                                                               |
 | `@auth/astro`          | Planned.                                                                                                                                                                               |
 | `@auth/nuxt`           | Planned. Community packages: [`@sidebase/nuxt-auth`](https://github.com/sidebase/nuxt-auth), [`@hebilicious/authjs-nuxt`](https://authjs-nuxt.pages.dev/)                              |
+| `@builder.io/qwik-auth`| Community package.                                                                                                                                                                               |
```
Comment (balazsorban44) [inline docs/docs/reference/index.md:24]: let's link to it

File: docs/docs/reference/index.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ title: Overview
 
 This section of the documentation contains the API reference for all the official packages under the `@auth/*` and `@next-auth/*` scopes.
 
-## Roadmap
+## Integrations
```
Comment (balazsorban44) [inline docs/docs/reference/index.md:7]: Need to update the link here:

https://github.com/nextauthjs/next-auth/blob/05bf014603bd8cc1f648b46771482d94ed7fd8c8/docs/docs/getting-started/introduction.mdx#L24

File: docs/docs/reference/index.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ title: Overview
...
 
-Here are the _state_ of planned and released packages under the `@auth/*` and `@next-auth/*` scope, as well as `next-auth`. This is not an exhaustive list, but the set of packages that we would like to focus on, to begin with.
+Here are the _state_ of planned and released packages under the `@auth/*` and `@next-auth/*` scope, as well as `next-auth`. It also includes community created and maintained packages. Packages listed as "Planned" are something we'd love help with! See the [help needed](#help-needed) section below.
```
Comment (balazsorban44) [inline docs/docs/reference/index.md:9]:
```suggestion
Here are the _state_ of planned and released integrations under the `@auth/*` and `@next-auth/*` scope, as well as `next-auth`. It also includes community created and maintained integrations. Integrations listed as "Planned" are something we'd love help with! See the [help needed](#help-needed) section below.
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**